lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml

```diff
@@ -2537,6 +2537,18 @@ definitions:
       ues_connected:
         type: integer
         format: int32
+      gps_altitude:
+        type: string
+      vendor:
+        type: string
+      model_name:
+        type: string
+      rf_state:
+        type: boolean
+      sw_version:
+        type: string
+      uptime:
+        type: string
```

Comment:
What format is this specified in? These details should be present in these `yml` definitions.